### PR TITLE
Map context layers

### PR DESCRIPTION
**Pull request by nerik**

### CODE REVIEW

File: scripts/components/map.component.js [hunk 1]
```diff
@@ -15,6 +17,50 @@ export default class {
...
+    const config = {'version': '1.0.1','layers':[{'user_name': 'wri-01','type': 'cartodb','options':{'sql':'SELECT * FROM ptw_grid_only_scoreover0','cartocss':'#layer{polygon-fill: #FF6600; polygon-opacity: 0.7; line-color: #FFF; line-width: 0; line-opacity:1;}','cartocss_version':'2.3.0','interactivity':['cartodb_id','grid_id']}}]};
+
+    fetch(`https://wri-01.cartodb.com/api/v1/map?config=${encodeURIComponent(JSON.stringify(config))}`)
+      .then(response => response.json())
+      .then((data) => {
+        console.log(data);
+        function getUrl(format) {
+          let url = 'https://' + username +
+            '.cartodb.com/api/v1/map/' + data.layergroupid +
+            '/0/{z}/{x}/{y}.' + format;
```
Comment (agnlez) [inline scripts/components/map.component.js]: Why you don't use ES6 templating here? :/

Comment (nerik) [reply scripts/components/map.component.js]: I dont know how but you commented on a deleted bit of code

Comment (agnlez) [reply scripts/components/map.component.js]: Ahh, I went commit by commit, probably you removed it later! My bad!

**Comment by agnlez** [comment]
After a rebase from `develop` and some sorcery, its seems it works. Let's wait @tiagojsag thoughts.

**Comment by tiagojsag** [comment]
- When I enable the layers, I get a lot of 400 status responses to tile requests:

```
{
errors: [
"Invalid or nonexistent map configuration token 'c176a73539feb243315ad0c0e6f30672'"
],
errors_with_context: [
{
type: "unknown",
message: "Invalid or nonexistent map configuration token 'c176a73539feb243315ad0c0e6f30672'"
}
]
}
```
- The layers side pane have a visible scroll bar, even though it does not fill the whole screen

**Comment by nerik** [comment]
@tiagojsag Can you check again plz?
Landcover layer not showing up yet because of https://basecamp.com/1756858/projects/12498794/todos/277660353#comment_473855518

**Pull request merged**